src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormatter.cs

```diff
@@ -88,5 +103,50 @@ private TextEdit[] MapEditsToHostDocument(RazorCodeDocument codeDocument, TextEd
 
             return actualEdits.ToArray();
         }
+
+        private async Task<TextEdit[]> FormatOnClientAsync(
+            RazorCodeDocument codeDocument,
+            Range projectedRange,
+            Uri uri,
+            FormattingOptions options,
+            CancellationToken cancellationToken)
+        {
+            var @params = new RazorDocumentRangeFormattingParams()
+            {
+                Kind = RazorLanguageKind.CSharp,
+                ProjectedRange = projectedRange,
+                HostDocumentFilePath = _filePathNormalizer.Normalize(uri.GetAbsoluteOrUNCPath()),
+                Options = options
+            };
+
+            var result = await _server.Client.SendRequest<RazorDocumentRangeFormattingParams, RazorDocumentRangeFormattingResponse>(
+                LanguageServerConstants.RazorRangeFormattingEndpoint, @params);
+
+            return result.Edits;
+        }
+
+        private async Task<TextEdit[]> FormatOnServerAsync(
+            RazorCodeDocument codeDocument,
+            Range projectedRange,
+            Uri uri,
+            FormattingOptions options,
+            CancellationToken cancellationToken)
+        {
+            var workspace = _projectSnapshotManagerAccessor.Instance.Workspace;
+            var cSharpOptions = workspace.Options
```

Comment:
Good question. This may have been a copy-paste from somewhere. I like `csharp` because `cSharp` kind of takes longer to type 😆. But I am willing to stick to whatever is most used.

Reply:
`csharp` +1

Reply:
+1 on `csharp`. Looks cleaner in my opinion.